contracts/escrow/Escrow.sol

```diff
@@ -1,77 +1,273 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.20;
+pragma solidity 0.8.20;
 
 import "../OgioParent.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "@chainlink/contracts/src/v0.8/shared/access/ConfirmedOwner.sol";
+import "@chainlink/contracts/src/v0.8/vrf/VRFV2WrapperConsumerBase.sol";
+import {SafeERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {Context} from "@openzeppelin/contracts/utils/Context.sol";
 
-contract Escrow is Ownable {
+contract Escrow is OgioService, VRFV2WrapperConsumerBase, ConfirmedOwner {
```

Comment:
VRFV2WrapperConsumerBase is needed to work with the Chainlink VRF random word generator